sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -174,7 +174,6 @@ export interface DataFeed {
     id: string;
     ingestionSettings: DataFeedIngestionSettings;
     isAdmin: boolean;
-    metricIds: string[];
```

Comment:
Is this intentional?

Reply:
Yes, ids alone aren't very helpful.  We were thinking metric ids are important, so we duplicated them from .schema.metrics.  But without other info, GUIDs are not really usable.